base/uk.ac.stfc.isis.ibex.ui/src/uk/ac/stfc/isis/ibex/ui/Utils.java

```diff
@@ -68,7 +69,12 @@ public static IWorkbenchPage getActivePage() {
             return null;
         }
 
-        return PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+        IWorkbenchWindow activeWorkbenchWindow = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
+        if (activeWorkbenchWindow == null) {
+            return null;
+        }
```

Comment:
Some documentation would be useful here. Yes we check for a null object but why only the WorkbenchWindow and not the Workbench itself. Why is returning null the correct behaviour if the window is null? Do we need to raise an exception? In these cases, it is clear what the code is doing, but documentation would help clarify why it's doing it.